draft-ietf-add-split-horizon-authority.xml

```diff
@@ -350,6 +361,28 @@
     </section>
     <!-- Validating -->
 
+    <section>
+      <name>Delegating DNSSEC across Split DNS Boundaries</name>
+      <t>We wish to enable DNSSEC validation of local DNS names without requiring
+      the local resolver to hold DNSSEC private keys that are valid for the parent
+      zone.  To support this configuration, parent zones <bcp14>MAY</bcp14> add a
+      "ds=..." key to the Verification Record whose value is the RDATA of a single
+      DS record, base64url-encoded. This DS record authorizes a DNSKEY whose Owner
+      Name is "resolver.arpa."</t>
+      <t>To validate DNSSEC, the client first fetches and validates the Verification
+      Record.  If it is valid and contains a "ds" key, the client <bcp14>MAY</bcp14>
+      send a DNSKEY query for "resolver.arpa." to the local encrypted resolver.  At
+      least one resulting DNSKEY RR <bcp14>MUST</bcp14> match the DS RDATA from the
+      Verification Record. All local resolution results for subdomains in this
```

Comment:
I think you mean match the "ds" key value in the Verification Record.

Reply:
Yes.  I've attempted to clarify this.